packages/nextjs/app/builders/0x6BF7d6b94282BD48ff458599aDafA268BcB009FF/page.tsx

```diff
@@ -0,0 +1,121 @@
+import { IconType } from "react-icons";
+import { FaEnvelope, FaGithub, FaLinkedin, FaMapMarkerAlt, FaTwitter } from "react-icons/fa";
+
+interface ProfileData {
+  name: string;
+  title: string;
+  bio: string;
+  avatar: string;
+  location: string;
+  email: string;
+  social: {
+    github: string;
+    twitter: string;
+    linkedin: string;
+  };
+}
+
+interface StyledIconProps extends React.HTMLAttributes<HTMLSpanElement> {
+  icon: IconType;
+  size?: number;
+  color?: string;
+}
+
+const StyledIcon: React.FC<StyledIconProps> = ({ icon: Icon, className, size, color, ...props }) => {
+  return (
+    <span className={`inline-block ${className}`} {...props}>
+      <Icon size={size} color={color} />
+    </span>
+  );
+};
+
+const ProfileCard: React.FC = () => {
```

Comment:
As more pages are added by other builders, consider giving this page a more descriptive name. Since it serves as a page in your Next.js application, you might also want to use the NextPage type. It extends React.FC and includes additional properties and methods that are specific to Next.js, such as getInitialProps.

Reply:
Pls change!